### PR TITLE
Example

### PR DESCRIPTION
**Pull request by shestakovg**
extended scroll view React Native component release

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -286,3 +286,5 @@ __pycache__/
 *.btm.cs
 *.odx.cs
 *.xsd.cs
+/windows/BlueJeans.Common.ExtendedScroll/Themes/Generic - Copy
```
Comment (abodalevsky) [inline .gitignore]: please, remove these files locally, don't add it to global place

File: Examples/Windows/Playground/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+using System.Runtime.InteropServices;
+using System.Windows;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Playground")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Microsoft")]
```
Comment (abodalevsky) [inline Examples/Windows/Playground/Properties/AssemblyInfo.cs]: definitely we're not Microsoft )

File: Examples/Windows/Playground/index.windows.animations.js [hunk 3]
```diff
@@ -0,0 +1,381 @@
+/**
```
Comment (abodalevsky) [inline Examples/Windows/Playground/index.windows.animations.js]: do you need this file

File: Examples/Windows/Playground/index.windows.js [hunk 4]
```diff
@@ -0,0 +1,131 @@
+/**
```
Comment (abodalevsky) [inline Examples/Windows/Playground/index.windows.js]: please remove all commented code

File: LICENSE [hunk 5]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```
Comment (abodalevsky) [inline LICENSE]: please remove this

File: index.windows.js [hunk 6]
```diff
@@ -0,0 +1,55 @@
+/**
```
Comment (abodalevsky) [inline index.windows.js]: seems we already have playground, looks like one of them are redundant...

File: package.json [hunk 7]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "react-native-bluejeans-components",
+  "version": "1.0.0",
```
Comment (abodalevsky) [inline package.json]: set please version to 0.0.1

**Comment by abodalevsky** [comment]
@shestakovg please ensure that your project has 4 configurations:
- Debug;
- DebugBundle;
- Release;
- ReleaseBundle;

File: Examples/Windows/Playground/index.windows.animations.js (hunk 3 above)
Comment (shestakovg) [reply Examples/Windows/Playground/index.windows.animations.js]: deleted

File: Examples/Windows/Playground/index.windows.js (hunk 4 above)
Comment (shestakovg) [reply Examples/Windows/Playground/index.windows.js]: done

File: windows/BlueJeans.Common.ExtendedScroll/ReactExtendedScrollManager.cs [hunk 8]
```diff
@@ -0,0 +1,705 @@
...
+    using System.Windows.Input;
+    using System.Windows.Media;
+
+    /// <summary>
+    /// ReactManager calss for ExtendedScrollViewer class
+    /// </summary>
+    public class ReactExtendedScrollManager : ViewParentManager<ExtendedScrollViewer>, IDisposable
+    {
+        /// <inheritdoc/>
+        public override string Name => "BlueJeansExtendedScrollManager";
```
Comment (abodalevsky) [inline windows/BlueJeans.Common.ExtendedScroll/ReactExtendedScrollManager.cs]: I believe name should be consistent with class name
Also I think that React is redundant.
Can we have name of class and all corespondent as _BlueJeansExtendedScrollManager_

File: windows/BlueJeans.Common.ExtendedScroll/ReactExtendedScrollManager.cs [hunk 9]
```diff
@@ -0,0 +1,705 @@
...
+        public override void AddView(ExtendedScrollViewer parent, DependencyObject child, int index)
+        {
+            if (index != 0)
+            {
+                throw new ArgumentOutOfRangeException(nameof(index), $"{nameof(ExtendedScrollViewer)} currently only supports one child.");
+            }
+
+            if (parent == null)
+            {
+                throw new ArgumentNullException("parent");
```
Comment (abodalevsky) [inline windows/BlueJeans.Common.ExtendedScroll/ReactExtendedScrollManager.cs]: `ArgumentNullException(nameof(parent))`
and the same bellow

File: windows/Playground/AppReactPage.cs [hunk 10]
```diff
@@ -18,13 +19,32 @@ namespace Playground
     internal class AppReactPage : ReactPage
     {
         /// <inheritdoc/>
-        public override string MainComponentName => "Playground";
+        public override string MainComponentName
```
Comment (abodalevsky) [inline windows/Playground/AppReactPage.cs]: please restore it, it should work  fine in VS 2015

File: package.json [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+  "main": "ExtendedScrollView.js",
+  "dependencies": {
+    "@bluejeans/react-native-windows": "^0.42.1-rc.23",
+    "chalk": "^1.1.3",
+    "glob": "^7.0.3",
+    "react": "~15.4.0-rc.4",
+    "react-native": "~0.42.0-rc.2",
+    "shelljs": "^0.7.0",
+    "uuid": "^2.0.1",
+    "xml-parser": "^1.2.1"
```
Comment (abodalevsky) [inline package.json]: please check if we really depends on all mentioned components

**Pull request merged**